/*
 * Copyright (c) 2023 Qualcomm Technologies, Inc.
 * All Rights Reserved.
 * Confidential and Proprietary - Qualcomm Technologies, Inc.
 */

syntax = "proto3";

package BluetoothHci;

// need this if we are using libprotobuf-cpp-2.3.0-lite
option optimize_for = LITE_RUNTIME;

/* Bluetooth HCI request */

message CloseReq
{
}

message InitializeReq
{
}

message AclDataReq
{
    bytes data = 1;
}

message CommandReq
{
    bytes command = 1;
}

message IsoDataReq
{
    bytes data = 1;
}

message ScoDataReq
{
    bytes data = 1;
}
